Gems/Atom/Bootstrap/Code/Source/BootstrapSystemComponent.cpp

```diff
@@ -410,25 +411,18 @@ namespace AZ
 
             void BootstrapSystemComponent::OnTick(float deltaTime, [[maybe_unused]] ScriptTimePoint time)
             {
-                m_simulateTime += deltaTime;
-                m_deltaTime = deltaTime;
-
-                // Temp: When running in the launcher without the legacy renderer
-                // we need to call RenderTick on the viewport context each frame.
                 if (m_viewportContext)
                 {
-                    AZ::ApplicationTypeQuery appType;
-                    ComponentApplicationBus::Broadcast(&AZ::ComponentApplicationBus::Events::QueryApplicationType, appType);
-                    if (appType.IsGame())
-                    {
-                        m_viewportContext->RenderTick();
-                    }
+                    m_viewportContext->SetFpsLimit(r_fps_limit);
```

Comment:
question (minor): Is it necessary to call this repeatedly? It might be the easiest thing but if there's a way to have this only called when the value is changed it might be nice?

Reply:
If I get some time I'll do the plumbing for this - it'll need a bus notification for the Editor bit to be able to listen to it, so it's a Whole Thing, but I shouldn't really add another branch to our critical path (albeit a prediction friendly one).

Reply:
Okay understood, thanks for the additional context, feels like it might be nice as it looks like `SetFpsLimit` is more than just an assignment